src/datamigration/azext_datamigration/manual/custom.py

```diff
@@ -0,0 +1,238 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+# pylint: disable=unused-argument
+# pylint: disable=line-too-long
+# pylint: disable=bare-except
+
+import ctypes
+import json
+import os
+import platform
+import subprocess
+import time
+import urllib.request
+from zipfile import ZipFile
+from knack.util import CLIError
+
+
+# -----------------------------------------------------------------------------------------------------------------
+# Assessment Command Implementation.
+# -----------------------------------------------------------------------------------------------------------------
+def datamigration_assessment(connection_string=None,
+                             output_folder=None,
+                             overwrite=False,
+                             config_file_path=None):
+
+    try:
+
+        if not platform.system().__contains__('Windows'):
```

Comment:
Can we move this validation to an Utility function, as it is being used in other commands also.